pep-0594.rst

```diff
@@ -167,33 +132,32 @@ audio processing.
    :header: "Module", "Deprecated in", "To be removed", "Added in", "Has maintainer?", "Replacement"
    :widths: 2, 1, 1, 1, 1, 2
 
-    aifc,3.8 (3.0\*),3.10,1993,**yes (inactive)**,\-
+    aifc,3.11 (3.0\*),3.13,1993,**yes (inactive)**,\-
     asynchat,**3.6** (3.0\*),3.10,1999,**yes**,asyncio_
     asyncore,**3.6** (3.0\*),3.10,1999,**yes**,asyncio_
-    audioop,3.8 (3.0\*),3.10,1992,**yes**,\-
-    binhex,3.8,3.10,1995,no,\-
-    cgi,3.8 (2.0\*\*),3.10,1995,no,\-
-    cgitb,3.8 (2.0\*\*),3.10,1995,no,\-
-    chunk,3.8,3.10,1999,no,\-
-    crypt,3.8,3.10,1994,**yes (inactive)**,"legacycrypt_, bcrypt_, argon2cffi_, hashlib_, passlib_"
+    audioop,3.11 (3.0\*),3.13,1992,**yes**,\-
+    binhex,3.11,3.13,1995,no,\-
+    cgi,3.11 (2.0\*\*),3.13,1995,no,\-
+    cgitb,3.11 (2.0\*\*),3.13,1995,no,\-
+    chunk,3.11,3.13,1999,no,\-
+    crypt,3.11,3.13,1994,**yes (inactive)**,"legacycrypt_, bcrypt_, argon2cffi_, hashlib_, passlib_"
     formatter,**3.4**,3.10,1995,no,\-
     fpectl,**3.7**,**3.7**,1997,n/a,\-
-    imghdr,3.8,3.10,1992,no,"filetype_, puremagic_, python-magic_"
-    imp,**3.4**,3.10,1990/1995,no,importlib_
-    macpath,**3.7**,**3.8**,1990,n/a,\-
-    msilib,3.8,3.10,2006,no,\-
-    nntplib,3.8,3.10,1992,no,\-
-    nis,3.8 (3.0\*),3.10,1992,no,\-
-    ossaudiodev,3.8,3.10,2002,no,\-
-    parser,**2.5**,**3.9**,1993,**yes**,"ast_, astroid_, lib2to3.pgen2"
-    pipes,3.8,3.10,1992,no,"subprocess_"
-    smtpd,"**3.4.7**, **3.5.4**",3.10,2001,**yes**,"aiosmtpd_"
-    sndhdr,3.8,3.10,1994,no,"filetype_, puremagic_, python-magic_"
-    spwd,3.8,3.10,2005,no,"python-pam_, simplepam_"
-    sunau,3.8 (3.0\*),3.10,1993,no,\-
-    telnetlib,3.8 (3.0\*),3.10,1997,no,"telnetlib3_, Exscript_"
-    uu,3.8,3.10,1994,no,\-
-    xdrlib,3.8,3.10,1992/1996,no,\-
+    imghdr,3.11,3.13,1992,no,"filetype_, puremagic_, python-magic_"
+    imp,**3.4**,3.12,1990/1995,no,importlib_
+    macpath,**3.7**,**3.11**,1990,n/a,\-
+    msilib,3.11,3.13,2006,no,\-
+    nntplib,3.11,3.13,1992,no,\-
+    nis,3.8 (3.0\*),3.13,1992,no,\-
+    ossaudiodev,3.11,3.13,2002,no,\-
+    pipes,3.1,3.13,1992,no,"subprocess_"
+    smtpd,"**3.4.7**, **3.5.4**",3.13,2001,**yes**,"aiosmtpd_"
+    sndhdr,3.11,3.13,1994,no,"filetype_, puremagic_, python-magic_"
+    spwd,3.11,3.13,2005,no,"python-pam_, simplepam_"
```

Comment:
```suggestion
    spwd,3.11,3.13,2005,no,"python-pam_"
```

Simplepam only had [one PyPI release in 2014](https://pypi.org/project/simplepam/#history) and [no commits since then(https://github.com/leonnnn/python3-simplepam).